packages/grafana-ui/src/components/ContextMenu/ContextMenu.story.tsx

```diff
@@ -18,39 +18,36 @@ const meta: ComponentMeta<typeof ContextMenu> = {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['renderMenuItems', 'renderHeader', 'onClose', 'children'],
+    },
   },
-};
-
-const menuItems = [
-  {
-    label: 'Test',
-    items: [
-      { label: 'First', ariaLabel: 'First' },
-      { label: 'Second', ariaLabel: 'Second' },
-      { label: 'Third', ariaLabel: 'Third' },
-      { label: 'Fourth', ariaLabel: 'Fourth' },
-      { label: 'Fifth', ariaLabel: 'Fifth' },
-    ],
+  args: {
+    x: 200,
+    y: 300,
+    focusOnOpen: true,
+    renderMenuItems: renderMenuItems,
   },
-];
+};
 
-const renderMenuItems = () => {
-  return menuItems.map((group, index) => (
-    <MenuGroup key={`${group.label}${index}`} label={group.label}>
-      {group.items.map((item) => (
-        <MenuItem key={item.label} label={item.label} />
-      ))}
-    </MenuGroup>
-  ));
+const renderHeader = (): React.ReactNode => {
+  return <h6>Menu</h6>;
 };
 
-export const Basic = () => {
-  return <ContextMenu x={10} y={11} onClose={() => {}} renderMenuItems={renderMenuItems} />;
+export const Basic: ComponentStory<typeof ContextMenu> = (args: ContextMenuProps) => {
+  return (
+    <ContextMenu
+      {...args}
+      onClose={() => action('onClose')('closed menu')}
+      renderMenuItems={renderMenuItems}
```

Comment:
Don't need this if it is passed as part of the `...args` now

Reply:
Good catch! I removed the renderMenuItems prop in the most recent commit as it as already included in args as stated.